src/main/java/org/folio/rest/impl/HridSettingsStorageAPI.java

```diff
@@ -0,0 +1,96 @@
+package org.folio.rest.impl;
+
+import java.util.Map;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.HridSettings;
+import org.folio.rest.jaxrs.resource.HridSettingsStorage;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.support.HridManager;
+import org.folio.rest.tools.utils.TenantTool;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class HridSettingsStorageAPI implements HridSettingsStorage {
+  private static final Logger log = LoggerFactory.getLogger(HridSettingsStorageAPI.class);
```

Comment:
```suggestion
  private static final Logger LOG = LoggerFactory.getLogger(HridSettingsStorageAPI.class);
```

Reply:
The general consensus is to use capitalization to express immutability. Loggers typically are not immutable. The Vert.x `Logger` lets you retrieve the underlying framework logger and modify it. This can be done from any class at any time.

Reply:
I usually use capitalized names for final fields. Did not think about mutability/immutability.

Reply:
Yeah, I didn't either until a few years ago. Google recommends not capitalizing static finals that are mutable.

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names